### PR TITLE
Implemented: Add additional job actions to job config component(#364tt29)

### PR DESCRIPTION
**Pull request by disha1202**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

Screenshot from 2022-08-26 15-29-14

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/hotwax/job-manager#contribution-guideline)

### CODE REVIEW

File: src/components/JobConfiguration.vue [hunk 1]
```diff
@@ -328,7 +416,17 @@ ion-list {
...
-  }  
+  }
+  .more-actions {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: end;
+    row-gap: var(--spacer-sm);
+    margin-top: var(--spacer-sm);
+  }
+  .more-actions > ion-item {
```
Comment (dt2patel) [inline src/components/JobConfiguration.vue:427]: we shouldn't specify ion item, it should just be all items in more-actions

**Review by dt2patel — CHANGES_REQUESTED** [review]
CSS comments

File: src/components/JobConfiguration.vue (hunk 1 above)
Comment (disha1202) [reply src/components/JobConfiguration.vue]: Made the requested change.

File: src/components/JobConfiguration.vue [hunk 2]
```diff
@@ -127,6 +155,9 @@ export default defineComponent({
   props: ["title", "status", "type"],
   computed: {
     ...mapGetters({
+      getEnumDescription: 'job/getEnumDescription',
+      getEnumName: 'job/getEnumName',
+      getPinnedJobs: 'user/getPinnedJobs',
```
Comment (adityasharma7) [inline src/components/JobConfiguration.vue:160]:
```suggestion
      pinnedJobs: 'user/getPinnedJobs',
```

File: src/components/JobConfiguration.vue [hunk 3]
```diff
@@ -70,13 +70,34 @@
...
+      {{ $t("Run now") }}
+    </ion-item>
+    <ion-item @click="copyJobInformation(currentJob)" button>
+      <ion-icon slot="start"  :icon="copyOutline" />
+      {{ $t("Copy details") }}
+    </ion-item>
+    <ion-item @click="updatePinnedJobs(currentJob?.systemJobEnumId)" button>
+      <ion-icon slot="start" :icon="pinOutline" />
+      {{ $t("Pin job") }}
+      <ion-checkbox slot="end" :checked="getPinnedJobs ? getPinnedJobs.includes(currentJob.systemJobEnumId) : false" />
```
Comment (adityasharma7) [inline src/components/JobConfiguration.vue:89]:
```suggestion
      <ion-checkbox slot="end" :checked="getPinnedJobs && getPinnedJobs.includes(currentJob.systemJobEnumId)" />
```

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

**Comment by disha1202** [comment]
> @disha1202 Please add the image of the work you have done in this PR.

Added the image.

File: src/components/JobConfiguration.vue (hunk 2 above)
Comment (disha1202) [reply src/components/JobConfiguration.vue]: Updated variable name.

File: src/components/JobConfiguration.vue (hunk 3 above)
Comment (disha1202) [reply src/components/JobConfiguration.vue]: Made the requested change.

**Review by azkyakhan — APPROVED** [review]

**Comment by azkyakhan** [comment]
LGTM

**Comment by azkyakhan** [comment]
LGTM

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**